### PR TITLE
Added License, Code of Conduct and Contribution Guidelines

### LINKED ISSUE #30: Link Code of Conduct, License and Contribution Guidelines to Readme File
**Is your feature request related to a problem? Please describe.**
Hey @Creatoon 
I would like to link above mentioned files to the readme file as they are present in the branch but not linked to the readme file.
Please assign it to me.
Thanks

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Shaishav0507**
Fixes #30 

Hey @Creatoon 
I have linked Code of Conduct, license and contribution guidelines in the readme file.
I have even added Project admin and created contributing.md file.
Please check and update.
Thanks.

#### Checklist

- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and improve it further
+
+To start contributing, follow the below guidelines: 
+
+🌟. Star🌟 the project to bookmark and appreciate the work.
+
+**0.**  Take a look at the existing [issues](https://github.com/Creatoon/Jar-web-app/issues) or create your own issues. Wait for the Issue to be assigned to you after which you can start working on it.
+
+**1.**  Fork [this](https://github.com/Creatoon/Jar-web-app/) repository.
```
Comment (Creatoon) [inline CONTRIBUTING.md]: Replace [this] with Repo name

File: readme.md [hunk 2]
```diff
@@ -51,10 +51,118 @@ JAR is web app that lets you connect to people around the world. You can create
...
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and improve it further
+
+To start contributing, follow the below guidelines: 
+
+🌟. Star🌟 the project to bookmark and appreciate the work.
+
+**0.**  Take a look at the existing [issues](https://github.com/Creatoon/Jar-web-app/issues) or create your own issues. Wait for the Issue to be assigned to you after which you can start working on it.
+
+**1.**  Fork [this](https://github.com/Creatoon/Jar-web-app/) repository.
```
Comment (Creatoon) [inline readme.md]: Replace [this] with [Jar-Web-App]

**Comment by Shaishav0507** [comment]
Ok I'll do it.

**Comment by Shaishav0507** [comment]
Done with the changes @Creatoon

**Comment by Creatoon** [comment]
@Shaishav0507  you are going great, just be regular.

**Comment by Shaishav0507** [comment]
Sure, thanks

**Pull request merged**